docs/examples/prometheus/kube-prometheus-stack-values-tls.yaml.example

```diff
@@ -0,0 +1,15 @@
+## Version > 1.3.0
```

Comment:
Should we add to the comment that this is the rabbitmq/cluster-operator version?
Otherwise, this might be confusing to users?